NewSpring/Rock
# Add Campus Events to NewSpring.cc

## DESCRIPTION

This PR adds lava templates that support displaying events on locations pages, as well as the main events page.

### How do I test this PR?

1. Pull down this branch.
2. Change `Initial Catalog` in your `web.connectionstrings.config` file to `Brian` for my database
3. Verify events are visible on every campus location page (/locations/columbia#events)
4. `See More Events` link should link to `/events` and be filtered for the campus that corresponds to the page you clicked `See More Events` on
5. Individual Events should link to their corresponding event detail pages
6. `/events` should display all events
7. Campus picker on `/events` should filter events by campus (and include events for all campuses)

## TODO


## REVIEW


> The purpose of PR Review is to _improve the quality of the software._
